README.md

```diff
@@ -8,33 +8,33 @@
   <a href="https://vercel.com/new/clone?repository-url=https://github.com/HsiangNianian/cWebDXCE" target="_blank">Deploy</a>
 </p>
 
-This Github repository is a dice application that includes both frontend and backend folders, using technology stacks such as Vue.js, axios, and Flask to achieve CRUD operations. Users can enter the name and number of faces of a dice on the page and add it to the list. Each dice has its unique ID, which can be used when the user clicks the delete or roll button.
+This repository contains a TTRPG dice Web app. It includes an user interface which has adopted Vue.js, axios, and Flask to enable CRUD operations. Users can add a dice instance to the dice list by entering the name and number of faces of a dice on the page. Each dice has its unique ID for operations such as roll, deletion, etc.

```

Comment:
**TTRPG**


- [ ] Typo to be corrected.

Reply:
Nope, TTRPG stands for **T**able**t**op **R**ole-**p**laying **G**ame, which is more widely accepted in recent years, because the term TRPG can be confused with **T**actical **R**ole **P**laying **G**ame.

Reply:
got it😀